### PR TITLE
Add Vercel recipie

### PR DESCRIPTION
**Pull request by AchoArnold**
Add the ability to run go fiber as a serverless app on vercel.

### CODE REVIEW

**Review by gaby — CHANGES_REQUESTED** [review]

File: vercel/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# [Vercel](https://vercel.com)
+
+Deploy and run a serverless go fiber application on vercel. With this recipie, you can also declare your routes using the fiber router.
+
+## Config
+
+Make sure to add the `vercel.json` file so the routing will work properly with vercel. This will rewrite all requests to the application to the `api/index.go` handler where where the router will take over.
```
[the remark this answers is not in the document]
Comment (AchoArnold) [reply vercel/README.md]: Fixed

**Review by gaby — APPROVED** [review]
Thanks!

**Comment by gaby** [comment]
CC @ReneWerner87 @efectn

**Comment by gaby** [comment]
@AchoArnold Can you add an entry under .github/dependabot.yml file. That way go.mod gets updated automatically.

**Comment by ReneWerner87** [comment]
- [ ] add an entry under .github/dependabot.yml file

**Comment by AchoArnold** [comment]
Done @gaby , @ReneWerner87

**Review by ReneWerner87 — APPROVED** [review]

**Pull request merged**